### ISSUE TITLE
Problem with wxLocale

### ISSUE THREAD

**Issue by lallulli** [lallulli/songpress#55]
```
#---- Notes ----#

Please provide additional information about the crash here

 

 

#---- System Information ----#

Songpress Version: 1.8.0

Operating System: Windows 10 (build 19042), 64-bit edition

Python Version: 3.8.7 (tags/v3.8.7:6503f05, Dec 21 2020, 17:43:54) [MSC v.1928 32 bit (Intel)]

wxPython Version: 4.1.1 msw (phoenix) wxWidgets 3.1.5

wxPython Info: (__WXMSW__, wxMSW, unicode, unicode-wchar, wx-assertions-on, phoenix, wxWidgets 3.1.5, autoidman, sip-4.19.24, build-type: release)

Python Encoding: Default=utf-8  File=utf-8

System Architecture: 32bit AMD64

Byte order: little

Frozen: True

#---- End System Information ----#

 

#---- Traceback Info ----#

*** Thu Apr  8 00:30:17 2021 ***

Traceback (most recent call last):

  File "main.py", line 32, in OnInit

  File "C:\Users\Luca Allulli\Documenti\dev\songpress\master\src\SongpressFrame.py", line 266, in __init__

  File "C:\Users\Luca Allulli\Documenti\dev\songpress\master\src\SongpressFrame.py", line 405, in AddTool

wx._core.wxAssertionError: C++ assertion "strcmp(setlocale(0, 0), "C") == 0" failed at ..\..\src\common\intl.cpp(1694) in wxLocale::GetInfo(): You probably called setlocale() directly instead of using wxLocale and now there is a mismatch between C/C++ and Windows locale.

Things are going to break, please only change locale by creating wxLocale objects to avoid this!

 

#---- End Traceback Info ----#
```

**Comment by lallulli** [comment]
i18n has been overhauled in Songpress 1.8.1. Now it uses wx.Locale instead of Python gettext.

**Comment by lallulli** [comment]
Fixed in Songpress 1.8.1.
